Samples/ResourceManagement/cs-winui/ClassLibrary/ClassLibrary.csproj

```diff
@@ -10,7 +10,8 @@
     <None Remove="ClassLibraryResources.resw" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.ProjectReunion" Version="0.8.1" />
+    <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.20348.19" />
```

Comment:
><PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.20348.19" /> [](http://example.com/codeflow?start=4&length=87)

is this necessary?

Reply:
Well, for this project, when updating the WindowsAppSDK nuget package to version 1.0 preview, it automatically adds the BuildTools. You mentioned in another comment that the PackageReference might not be necessary, and after trying it out without the BuildTools line, it does work. However, since VS insists in adding it, do you think I should still remove it?

Reply:
If VS adds it, leave it there. I didn't know you manually removed BuildTools reference in your previous iterations.

Reply:
I'll add them back